lib/rtcpeerconnection/chrome.js

```diff
@@ -32,7 +32,10 @@ function ChromeRTCPeerConnection(configuration, constraints) {
 
   EventTarget.call(this);
 
-  var newConfiguration = Object.assign({}, configuration);
+  // NOTE(mhuynh): See
+  // https://groups.google.com/forum/#!searchin/discuss-webrtc/sdpSemantics%7Csort:date/discuss-webrtc/nDdDqIBtFBM/hlf_Z9EnCgAJ
```

Comment:
It may be better to change this link to https://webrtc.org/web-apis/chrome/unified-plan/